oak/server/rust/oak_runtime/src/runtime/proxy.rs

```diff
@@ -61,6 +62,7 @@ impl RuntimeProxy {
         application_configuration: &ApplicationConfiguration,
         grpc_configuration: &GrpcConfiguration,
     ) -> RuntimeProxy {
+        let (introspection_event_sender, _) = broadcast::channel(100);
```

Comment:
Please make this a const with a meaningful name, and ideally also some comment :)

Reply:
Done! Just to make sure understand you correctly, the suggestion is to make the argument (`100`) of the `broadcast::channel` function as const? I also added a comment to explain that value.